website/content/docs/bctl-reference/apply.md

```diff
@@ -17,5 +17,5 @@ bctl apply [flags]
 | Flag | Description | Default |
 | ---- | ----------- | ------- |
 | `-h, --help` | Display the help for apply |
-| `-f, --file` | The path to the blueprint file to apply | `./boundless.yaml` |
-| `operator-uri` | The URL or path of the boundless-operator manifest to use | `https://raw.githubusercontent.com/mirantiscontainers/boundless/main/deploy/static/boundless-operator.yaml` |
+| `-f, --file` | The path to the blueprint file to apply | `./blueprint.yaml` |
+| `operator-uri` | The URL or path of the blueprint-operator manifest to use | `The URL or path of the blueprint-operator manifest to use | `https://raw.githubusercontent.com/mirantiscontainers/boundless/main/deploy/static/boundless-operator.yaml` |
```

Comment:
This no longer exists. I thought it was hidden in the help output and not called out in the docs. We can do a followup PR if you don't want to remove it here.